tests/integration/targets/ntnx_acps/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
is this the project used created during setup ?

Reply:
yes